### ISSUE TITLE
[iOS] Minute interval doesn't work with mode 'time'

### ISSUE THREAD

**Issue by dansiemens** [henninghall/react-native-date-picker#132]
**Describe the bug**
When the `mode` is set to `'time'`, the `minuteInterval` prop has no effect (iOS only). Notably, it works when mode is `'datetime'`. 

**Expected behavior**
Minute interval prop should work on iOS when mode is `time`

**To Reproduce**
Add example code that reproduces the behavior. 
```javascript
export default class App extends Component {

  state = { date: new Date() }

  render = () =>
    <DatePicker
      mode='time'
      minuteInterval={15}
      date={this.state.date}
      onDateChange={date => this.setState({ date })}
    />

}
```

**Smartphone (please complete the following information):**
 - OS: iOS 12.1
 - React Native version:  0.61.4
 - react-native-date-picker version: 2.7.0

**Comment by henninghall** [comment]
Hi, thanks for reporting this. I have tried but I don't manage to reproduce it. Could you please double-check that you actually are running the exact code above? Which iPhone model are you using? 

My result with your code sample and software versions. 

<img width="342" alt="Screenshot 2019-12-27 at 23 28 21" src="https://user-images.githubusercontent.com/1652607/71534688-b7e1f880-2900-11ea-9d30-36d5af7ad61a.png">

**Comment by xhirazi** [comment]
+1

**Comment by henninghall** [comment]
Bump @dansiemens 🙂Did you have any additional information?

**Comment by dansiemens** [comment]
@henninghall Tried to reproduce after upgrading to 2.7.2 and i can no longer reproduce the issue. Found the issue only on 2.7.0, running Xcode 10.1, reproduced in 12.1 simulator and on iPhone 6 and iPhone X both running 12.x.

**Comment by henninghall** [comment]
Thanks a lot! 
@xhirazi Could you try upgrading to version 2.7.2 or above and see if that helps?

**Comment by henninghall** [comment]
Found a way to reproduce this, and have a fix ready. Will merge https://github.com/henninghall/react-native-date-picker/issues/137 and release a new version shortly.

**Comment by henninghall** [comment]
Released in v2.7.4
